core/package.json

```diff
@@ -51,27 +51,28 @@
     "bundle-develop-ie11": "npm run bundle-ie11 -- -d --watch",
     "test": "babel-node ./node_modules/nyc/bin/nyc.js mocha -- --recursive test",
     "test:watch": "npm run test -- --watch",
-    "bundlesize": "npm run bundle && bundlesize"
+    "bundlesize": "npm run bundle && bundlesize",
+    "bundlesizeOnly": "bundlesize"
   },
   "bundlesize": [
     {
       "path": "./public/luigi.js",
-      "maxSize": "100 kB",
+      "maxSize": "350 kB",
```

Comment:
Is this raise really necessary? I guess you tried with `npm run bundle-develop` .. please check with `npm run bundle`. Bundlesize in CI is always using production build. Also for other sizes, I guess they are fine as they have bene.

Reply:
I've tried with `npm run bundle` and the result for luigi.js is around 297KB. So, I gave it a little bit more space, counting it would be enough for the next few months. Any suggestions are welcome. 

With `npm run bundle-develop`, the result for luigi.js is 1.9MB.

Reply:
Well, it has been a long time since I really looked at the bundlesize and thought we always enforced it and it stays below 100kb. It looks like we have built tons of new stuff which brought in code size ;-) So let's leave it for now thanks.